analysis/tests/src/CompletionPattern.res

```diff
@@ -191,3 +191,13 @@ let s = (true, Some(true), [false])
 
 // switch b { | #one | #three({test: true}, true | )  }
 //                                                ^com
+
+type variantWithInlineRecord = One({thing: bool, otherThing: option<bool>})
+
+let r: variantWithInlineRecord = One({thing: true, otherThing: None})
+
+// switch r { | One()}
```

Comment:
Are these tests for this PR?
They look like good tests anyhow.

Reply:
Nope, good catch. I'll add them in a separate PR for inline records.